modules/swagger-codegen/src/main/resources/typescript-angular2/api.mustache

```diff
@@ -1,14 +1,16 @@
 {{>licenseInfo}}
-import {Http, Headers, RequestOptionsArgs, Response, URLSearchParams} from '@angular/http';
-import {Injectable, Optional} from '@angular/core';
-import {Observable} from 'rxjs/Observable';
-import * as models from '../model/models';
-import 'rxjs/Rx';
+import { Injectable, Optional }                              from '@angular/core';
+import { Http, Headers, URLSearchParams }                    from '@angular/http';
+import { RequestMethod, RequestOptions, RequestOptionsArgs } from '@angular/http';
+import { Response, ResponseContentType }                     from '@angular/http';
```

Comment:
Breaking change. ResponseContentType is new from Angular RC5 this means that you will be forced to upgrade to Angular RC5 when using this change which makes it breaking. Because Angular RC5 breaks against RC2 in a lot of ways.